extensions/apps/extensions/src/components/pages/info-page/sub-pages/dev-info-page.tsx

```diff
@@ -144,12 +148,10 @@ export const DevInfoPage = (props: DevInfoPageProps) => {
           {appsReq.error && (
             <>
               <Divider />
-              <ErrorLoader
-                noWrapperCard={true}
-                type="list-not-available"
-                title={`${t('Uh-oh')}!${t("We couldn't load the extension list")}!`}
-                details={`${t('It seems there is a problem retreving the list of extensions')}. ${t('Please try again later')}!`}
-              />
+              <ErrorLoader type="list-not-available">
```

Comment:
wherever there was noWrapperCard prop used you need to translate it to `border-none` using className

Reply:
you need the same changes applied elsewhere in this PR where `noWrapperCard` was used